src/Chunk.cpp

```diff
@@ -3050,15 +3050,15 @@ void cChunk::BroadcastSpawnEntity(cEntity & a_Entity, const cClientHandle * a_Ex
 
 
 
-void cChunk::BroadcastThunderbolt(int a_BlockX, int a_BlockY, int a_BlockZ, const cClientHandle * a_Exclude)
+void cChunk::BroadcastThunderbolt(Vector3i a_Block, const cClientHandle * a_Exclude)
```

Comment:
`a_BlockPos`.
Is thunderbolt block-aligned? Shouldn't it be a `Vector3d` instead? In such a case, `a_Position` would be more adequate.

Reply:
In Vanilla Minecraft, it is not block aligned, but in Cuberite it appears to be block aligned all the way to the protocol.